addons/l10n_nl/migrations/3.1/post-migrate_update_taxes.py

```diff
@@ -3,4 +3,14 @@
 
 
 def migrate(cr, version):
+    cr.execute(
+        r"""
+        UPDATE ir_model_data
+           SET name = name || '_producten'
+         WHERE module = 'l10n_nl'
+           AND model = 'account.tax'
+           AND name ~ '^\d+_btw_x0$'
```

Comment:
Whoops, actually..
the `x` should be `X` (uppercase)

Reply:
Changes Done.